packages/renderer/src/lib/preferences/PreferencesAuthenticationProvidersRendering.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+  import { authenticationProviders } from '../../stores/authenticationProviders';
+  import DropdownMenu from '../ui/DropdownMenu.svelte';
+  import DropdownMenuItem from '../ui/DropDownMenuItem.svelte';
+  import { faUser } from '@fortawesome/free-solid-svg-icons';
+</script>
+  
+  <div class="flex w-full flex-col p-2 bg-zinc-800">
```

Comment:
It may have been released after you started this PR, but please use SettingsPage component here (see other pages) to have a consistent page header.